modules/i2c/i2cs.h

```diff
@@ -0,0 +1,50 @@
+/** @defgroup i2c I2C
+ * @brief I2C module
+ */
+//@{
+/**
+ * @file
+ * @brief I2C definitions
+ */
+#ifndef I2CS_H__
+#define I2CS_H__
+
+#include <avr/io.h>
+#include <avarix/intlvl.h>
+#include <stdint.h>
+#include "i2c_config.h"
+
+/* Transaction status defines.*/
+#define I2CS_STATUS_READY                0
+#define I2CS_STATUS_BUSY                 1
+
+/* Transaction result enumeration */
+typedef enum {
+  I2CS_RESULT_UNKNOWN            = 0,
```

Comment:
Don't align commas (and equal).

Also, why using `(0x01<<0)` and not simply 1?